crates/meilisearch-types/src/keys.rs

```diff
@@ -365,6 +365,12 @@ pub enum Action {
     #[serde(rename = "*.get")]
     #[deserr(rename = "*.get")]
     AllGet,
+    #[serde(rename = "webhooks.get")]
+    #[deserr(rename = "webhooks.get")]
+    WebhooksGet,
+    #[serde(rename = "webhooks.update")]
+    #[deserr(rename = "webhooks.update")]
+    WebhooksUpdate,
```

Comment:
Inconsistency between the PRD and the implementation https://www.notion.so/meilisearch/Usage-for-webhook-API-23f4b06b651f8054b6fed3ca02641770

Should we update the code or the PRD?